include/cantera/zeroD/ConnectorFactory.h

```diff
@@ -0,0 +1,91 @@
+//! @file ConnectorFactory.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CONNECTOR_FACTORY_H
+#define CONNECTOR_FACTORY_H
+
+#include "cantera/base/FactoryBase.h"
+#include "cantera/zeroD/Connector.h"
+
+namespace Cantera
+{
+
+class FlowDevice;
+class WallBase;
+
+//! Factory class to create Connector objects.
+//!
+//! This class is mainly used via the newConnector() function, for example:
+//!
+//! ```cpp
+//!     shared_ptr<Connector> mfc = newConnector("Valve", r0, r1, "my_valve");
```

Comment:
```suggestion
//!     shared_ptr<Connector> valve = newConnector("Valve", r0, r1, "my_valve");
```
And again in the group description below.